src/Application/Actions/CreatePasswordResetToken.php

```diff
@@ -28,18 +29,20 @@ public function __construct(
         parent::__construct($logger);
     }
 
-    protected function action(): Response
+    protected function action(Request $request, array $args): Response
```

Comment:
This so much clearer, nice one!

Reply:
(hmm actually that's a lie, noticed this is protected so we always have to go via invoke)

Reply:
I see